src/components/organisms/news/news.scss

```
@@ -0,0 +1,66 @@
+@use "../../variables" as *;
+
+.news{
+
+.contents {
+  padding-left: 0px;
+  margin-bottom: 30px;
+
+  a{
+    text-decoration: none;
+  }
+    
+    article {
+    background-color: $white;
+    display: flex;
+    flex-direction: column;
+    padding: 20px;
+    height: 85px;
+    border-top: 1px solid $gray_gainsboro;
+    border-left: 1px solid $gray_gainsboro;
+    border-right: 1px solid $gray_gainsboro;
+    cursor: pointer;
+
+    time {
+      margin: 0px 0px 10px;
+      font-size: 12px;
+      line-height: 12px;
+      color: $gray_nobel;
+    }
+
+    h2 {
+      font-size: 16px;
+      font-weight: 400;
+      line-height: 23px;
+      margin-bottom: 0px;
+      overflow: hidden;
+      white-space: nowrap;
+      text-overflow: ellipsis;
+      transition: 0.2s ease;
+    }
+
+    article:hover {
+      color: $orange_bistre;
+    }
+    
+  }
+
+  article:last-child {
+    border-bottom: 1px solid $gray_gainsboro;
+  }
```

この書き方だとarticle:last-childつまり、articleの中の全てのh2に対してborder-bottomが効いてしまっているのでボーダーが二重になっている箇所が確認できると思います。
今回は` <div class="contents">`の:last-childにスタイルを効かせたいので以下が適切だと思います（間違っているいるかもしれないので試してみてください）
```suggestion
  :last-child {
    border-bottom: 1px solid $gray_gainsboro;
  }
```